pkg/crc/constants/constants.go

```diff
@@ -38,6 +38,9 @@ const (
 
 	VSockGateway = "192.168.127.1"
 	VsockSSHPort = 2222
+
+	BaseDomain = ".crc.testing"
+	AppsDomain = ".apps-crc.testing"
```

Comment:
We usually take this from the bundle info instead having it in constants.

Reply:
I will open a follow up to put everything in common. These constants are also used in crc setup for the dns part.

Reply:
The end-goal is to kill them, not to make use of them everywhere :)